tests/unit/test_client.py

```diff
@@ -430,10 +473,11 @@ def test_oauth2_exceed_max_attempts(attempts, sample_post_response_data):
 
 @pytest.mark.parametrize("header,error", [
     ("", "Error: header WWW-Authenticate not available in the response."),
-    ('Bearer"', 'Error: header info didn\'t have x_redirect_server'),
-    ('x_redirect_server="redirect_server", x_token_server="token_server"', 'Error: header info didn\'t match x_redirect_server="redirect_server", x_token_server="token_server"'),  # noqa: E501
+    ('Bearer"', 'Error: header info didn\'t have x_token_server'),
+    ('x_redirect_server="redirect_server", x_token_server="token_server"',
+     'Error: header info didn\'t match x_redirect_server="redirect_server", x_token_server="token_server"'),
+    # noqa: E501
```

Comment:
the `noqa` was here to prevent wrapping across lines. Revert the wrapping.

Reply:
missed that bit. Thanks!